src/a.jl

```diff
@@ -27,6 +28,44 @@ function axisar(axis, angle)
     permutedims(r)
 end
 
+"""
+    azlcpo(method, target, et, abcorr, azccw, elplsz, obspos, obsctr, obsref)
+
+Return the azimuth/elevation coordinates of a specified target relative to an "observer," 
+where the observer has constant position in a specified reference frame. 
+The observer's position is provided by the calling program rather than by loaded SPK files.
+
+### Arguments ###
+
+- `method`: Method to obtain the surface normal vector. The only choice currently supported is `"ELLIPSOID"`
```

Comment:
To have a more "julian" API it would make sense to make the `method`, `azccw`, and `elplsz` parameters optional keyword argument.

Reply:
Sure, I can do this. For consistency, I would put `azccw` and `elplsz` as optional keyword arguments also in `azlrec`, `recazl`, `drdazl` and `dazldr`.